system/application/views/widgets/mediaelement/jquery.mediaelement.js

```diff
@@ -5488,6 +5504,63 @@ function YouTubeGetID(url){
 		}
 
 	}
+  
+  /**
+	 * View for IIIF manifest (Mirador) content.
+	 * @constructor
+	 *
+	 * @param {Object} model		Instance of the model.
+	 * @param {Object} parentView	Primary view for the media element.
+	 */
+	jQuery.MiradorObjectView = function(model, parentView) {
+
+		var me = this;
+
+		this.model = model;  					// instance of the model
+		this.parentView = parentView;   		// primary view for the media element
+		this.isLiquid = true;					// media will expand to fill available space
+
+		/**
+		 * Creates the video media object.
+		 */
+		jQuery.MiradorObjectView.prototype.createObject = function() {
+
+			var approot = $('link#approot').attr('href');
+
+			this.mediaObject = $( '<div class="mediaObject" id="mirador"></div>' ).appendTo( this.parentView.mediaContainer );
+      
+      var miradorInstance = Mirador.viewer({
+        id: 'mirador',
+        windows: [
+          { manifestId: this.model.node.current.sourceFile }
+        ]
+      });
+
+			this.parentView.layoutMediaObject();
+			this.parentView.removeLoadingMessage();
+
+			return;
+		}
+
+		// These functions are basically irrelevant for this type of media
```

Comment:
Do we need jQuery.MiradorObjectView.prototype.play and those other functions?

IIIF will eventually expand to include audio/video content, but not for quite some time. Not within our scope.

Reply:
I went back and rewatched the zoom call we had with Erik and Craig, and from what I understood those methods need to be present, but set to null or empty functions if you don't want the functionality. I modeled it on the Deep Zoom player that the've set up.